build/attendee/index.html

```diff
@@ -19,6 +19,14 @@
 
 <body>
   <div id="app"></div>
+
+  <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
```

Comment:
I mean, sort of? On the other hand, we now have to manage dependencies in about four places.